virttest/qemu_devices/qdevices.py

```diff
@@ -1560,9 +1560,11 @@ def get_supported_options(self, backend):
             special_opts.append("signal")
 
         elif backend in ["socket"]:
-            common_opts += ["server", "nowait", "reconnect"]
+            common_opts += ["server", "wait", "reconnect"]
             special_opts = ["host", "port", "to", "ipv4",
                             "ipv6", "nodelay", "path"]
+            # TODO: The 'delay' option for socket character devices is
+            #  deprecated from qemu 6.0, so might be removed from here later
```

Comment:
Could you explain more about the reason that do not modify `delay` together ?

Reply:
@yanan-fu It's still supported on the old version, here just keep the option, but not specify it by default, so if someone need to test it, still can configure it in configuration file. But maybe should remove it someday when no one need to test qemu version older than 6.0.

Reply:
@vivianQizhu `delay` and `nodelay` are 2 different options, you are right that `delay` will be removed, while will keep use `nodelay` boolean.